src/pages/Game/index.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import styled from "styled-components";
+
+import Header from "../../components/Header/index";
+import BlockCombinator from "../../components/BlockCombinator/index";
+import Map from "../../components/Map/index";
```

Comment:
```suggestion
import Header from "../../components/Header";
import BlockCombinator from "../../components/BlockCombinator";
import Map from "../../components/Map";
```
파일명이 index.js 이면, import시 폴더경로까지만 작성해줘도 되는걸로 알고있어요.
경로까지만 작성해준다면 불필요한 코드를 줄일 수 있으니 더 좋아보입니다!